apps/bcc3d.ca/assets/js/request.js

```diff
@@ -1,29 +1,60 @@
 ---
 ---
+const REGIONS_API_ENDPOINT = "https://www.qa.bcc3d.ca/api/v1/region";
```

Comment:
This should be templated, like `{{ site.endpoints.hcp_submit }}`

Reply:
Where do we keep reference of these endpoints?

Reply:
`config.yml` at the top-level directory, then when you run `sh build.sh` it templates these values into `apps/bcc3d.ca/_config.yml` which is read by the jekyll build process

Reply:
have a look at `templates/apps/bcc3d.ca/_config.yml`, anything in the `templates` folder will have the values in `config.yml` (the top level one) interpolated into it as the first part of the build process in `build.sh`

Reply:
My blocker on testing this was the fact that on the local build the flask API endpoints were returning 404s, potentially due to either the nginx proxy_pass directive or the flask app configuration.

I will try again today and report any issues I see. I'll fix up the API code if needed (and if I can figure it out)